source/deployment/sso-ldap.md

```diff
@@ -49,9 +49,35 @@ Notes:
 
 Organizations using multiple domains can integrate with Mattermost using a "Forest" configuration to bring together multiple domains. Please see [Forests as Collections of Domain Controllers that Trust Each Other](https://technet.microsoft.com/en-us/library/cc759073%28v=ws.10%29.aspx?f=255&MSPPError=-2147217396) for more information.
 
-### Troubleshooting
+### Troubleshooting / FAQ
 
-The following are troubleshooting suggestions on common error messages and issues.
+The following are frequently asked questions and troubleshooting suggestions on common error messages and issues.
+
+##### When I first set up and synchronize LDAP, are the users automatically created in Mattermost? 
+
+No, the users are created on first log in. 
+
+##### If I want to add people to channels, can I pre-create users somehow? 
+
+Yes, using the [bulk import tool](https://docs.mattermost.com/deployment/bulk-loading.html#bulk-loading-data).
+
+##### How does deactivating users work? 
+
+- Deleting Users: Deleting a user from LDAP will deactivate them from Mattermost on the next sync. 
+- Disabling Users: Depends on the LDAP server. Some may deactivate the user in Mattermost, for others, such as Active Directory, you will need to specify with the user filter. 
```

Comment:
This doesn't have the tone of the rest of our docs, it sounds like a chat message response

Reply:
Updated